tpl_lessallrounder/index.php

```diff
@@ -138,7 +138,7 @@
 					<jdoc:include type="modules" name="position-7" style="allroundersidebar" />
 				</div>
 			<?php endif; ?>
-			<?php if ($this->countModules('position-8')) : ?>
+			<?php if (($this->countModules('position-8')) and ($params->get('rightSBtoBottom') == false) ) : ?>
```

Comment:
Since you have a default value of `1` in the XML, it needs to have a default value here as well.
Also we can make the check a simple boolean check with `!`. It does the same but I prefer it this way as I find it easier to read. The added parantheses are not needed by the way.
So please use:
`if ($this->countModules('position-8') and !$params->get('rightSBtoBottom', 1)) :`